pkg/util/taints/taints.go

```diff
@@ -131,7 +131,8 @@ func ReorganizeTaints(node *v1.Node, overwrite bool, taintsToAdd []v1.Taint, tai
 	newTaints := append([]v1.Taint{}, taintsToAdd...)
 	oldTaints := node.Spec.Taints
 	// add taints that already existing but not updated to newTaints
-	added := addTaints(oldTaints, &newTaints)
+	addedTaints, added := addedTaints(oldTaints, newTaints)
+	newTaints = append(newTaints, addedTaints...)
```

Comment:
The PR LGTM except the naming. (It's just my understanding, and @endocrimes do you think so?)

- newTaints -> addingTaints
- oldTaints -> oldTaints
-  addedTaints -> existedTaints

```suggestion
	existedTaints, added := addedTaints(oldTaints, append([]v1.Taint{}, taintsToAdd...))
	newTaints = append(existedTaints, newTaints...)
```